echopype/utils/prov.py

```diff
@@ -59,4 +64,11 @@ def source_files_vars(source_paths: Union[str, List[Any]]) -> Dict[str, Tuple]:
         ),
     }
 
-    return source_files_var
+    source_files_coord = {
+        "filenames": (
+            "filenames",
+            list(range(len(source_files))),
+            {"long_name": "Source filenames index"},
```

Comment:
```suggestion
            {"long_name": "Index for data source file names"},
```
I made this small change because it looks like you are maybe referring to the variable `source_filenames`. I wanted to refrain from doing this because we plan on using this coordinate in issue #677 for another variable. Feel free to change it, if your think there is a better long name.

Reply:
Good point and good suggestion for a `long_name`. But I'll deal with it in the next PR, the one that actually implements #677. I think it'll be best to finalize that long_name string in that context.